### PR TITLE
SEO optimizations, add first Netlify form, minor wording fixes

### LINKED ISSUE #153: Set up forms through Netlify with Slack notifications
## Is your feature request related to a problem?

Nope.

## Describe the solution you'd like

Netlify Forms to onboard new members and track leads for sponsors and speakers, etc.

Every new response will send a Slack message to a certain channel like below.

Slack + Netlify

## Describe alternatives you've considered

Google Forms currently being used and requires more manual labor

## Additional context

https://www.netlify.com/docs/form-handling/

https://www.netlify.com/blog/2016/07/18/shiny-slack-notifications-from-netlify/

Related to #95 and #54

### PR DESCRIPTION
**Pull request by fvcproductions**
Signed-off-by: Frances Coronel <hello@fvcproductions.com>

- fixes #153
- optimizes SEO
- minor wording fixes 

---

> ✅️ By submitting this PR, I have verified the following

- [x] Checked to [see if a similar PR has already been opened](https://github.com/techqueria/website/pulls) 🤔️
- [x] Reviewed the [contributing guidelines](https://github.com/techqueria/website/blob/master/.github/CONTRIBUTING.md) 🔍️
- [x] Added my name to the bottom of the list under the **Contributors** section in the [README.md](https://github.com/techqueria/website/blob/master/README.md) with a link to my personal website or GitHub profile 👥️

### CODE REVIEW

File: site/layouts/partials/head/meta-custom.html [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+{{ end }}
+<!-- Twitter Labels -->
+{{ if .Params.tags }}
+<meta name="twitter:label2" value="Tags 📌️">
+<meta name="twitter:data2" value="{{ delimit .Params.tags ", " }}">
+{{ end }}
+{{ end }}
+<!-- Keywords -->
+{{ if or (.Params.categories) (.Params.tags) }}
+<meta itemprop="keywords" content="{{ if .Params.categories }}{{ delimit .Params.categories ", " }}{{ end }}{{ if .Params.tags }}{{ delimit .Params.tags ", " }}{{ end }}techqueria">
```
Comment (fvntr) [inline site/layouts/partials/head/meta-custom.html]: these params are in config toml?

File: site/content/contact.md [hunk 2]
```diff
@@ -13,6 +13,43 @@ description: Get in touch with Latinx in Tech!
 
 Use the command `/admin` in [our Slack workspace](/slack/).
 
+### Contact Form
```
[the remark this answers is not in the document]
Comment (fvcproductions) [reply site/content/contact.md]: u sure can!

File: site/layouts/partials/head/meta-custom.html (hunk 1 above)
Comment (fvcproductions) [reply site/layouts/partials/head/meta-custom.html]: no those only apply to blog posts for the future - they could have tags or categories

File: src/sass/_overrides.scss [hunk 3]
```diff
@@ -20,6 +20,15 @@
   line-height: 1;
 }
 
+form button,
+form input,
+form label,
+form pre code,
+form span,
+form textarea {
+  font-size: inherit !important;
```
[the remark this answers is not in the document]
Comment (fvcproductions) [reply src/sass/_overrides.scss]: has to be :/

**Pull request merged**